code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -269,11 +242,11 @@
 
 
 /datum/admins/proc/makeAliens()
-	new /datum/round_event/alien_infestation{spawncount=3}()
+	new /datum/round_event/ghost_role/alien_infestation{spawncount=3}()
```

Comment:
Could you make these an argument please, this notation really ticks me off outside of map definitions.